manifests/plugin/gutterball.pp

```diff
@@ -0,0 +1,9 @@
+# gutterball plugin
+class katello::plugin::gutterball{
+    Class[ 'certs' ] ->
```

Comment:
Nitpick - indentation is off with respect to other files (2 spaces instead of 4)